tests/unit/cluster/no-failover-option.tcl

```diff
@@ -0,0 +1,66 @@
+# Check that the no-failover option works
+
+source tests/support/cluster.tcl
+
+start_cluster 5 5 {tags {external:skip cluster}} {
```

Comment:
```suggestion
start_cluster 3 3 {tags {external:skip cluster}} {
```
We could probably cut this down to bare minimum for quorum. The test takes 17 seconds, I believe most of it is because of starting and meeting. Not a strong opinion.